app/src/main/java/tk/wasdennnoch/androidn_ify/extracted/systemui/qs/QSDetail.java

```diff
@@ -172,6 +174,11 @@ public void setExpanded(boolean qsExpanded) {
     private void updateDetailText() {
         //mDetailDoneButton.setText(R.string.quick_settings_done);
         //mDetailSettingsButton.setText(R.string.quick_settings_more_settings);
+        int buttonPadding = ResourceUtils.getInstance(mContext).getDimensionPixelSize(R.dimen.qs_detail_button_padding);
+        mDetailDoneButton.setBackground(ResourceUtils.getInstance(mContext).getDrawable(R.drawable.qs_btn_borderless_rect));
+        mDetailSettingsButton.setBackground(ResourceUtils.getInstance(mContext).getDrawable(R.drawable.qs_btn_borderless_rect));
+        mDetailDoneButton.setPadding(buttonPadding,buttonPadding,buttonPadding,buttonPadding);
+        mDetailSettingsButton.setPadding(buttonPadding,buttonPadding,buttonPadding,buttonPadding);
```

Comment:
Would have probably been better to just declare a variable instead of calling getDrawable 2 times, but I forgot to do that. Feel free to add if you wish to.